File: snippets/rust/src/static_channel_backup.rs
use anyhow::Result;
use breez_sdk_core::*;

async fn retrieve_backup_files() -> Result<()> {
    // ANCHOR: static-channel-backup
    let backup_data = BreezServices::static_backup(StaticBackupRequest {
        working_dir: "<working directory>".into(),
    })?;
    // ANCHOR_END: static-channel-backup

    Ok(())
}

File: snippets/csharp/StaticChannelBackup.cs
using Breez.Sdk;

public class StaticChannelBackupSnippets
{
  public void StaticChannelBackup(BlockingBreezServices sdk)
  {
    // ANCHOR: static-channel-backup
    try
    {
      var backupData = BreezSdkMethods.StaticBackup(
          new StaticBackupRequest("<working directory>"));
    }
    catch (Exception)
    {
      // Handle error
    }
    // ANCHOR_END: static-channel-backup
  }
}
